pkgs/development/haskell-modules/configuration-nix.nix

```
@@ -359,6 +359,13 @@ self: super: builtins.intersectAttrs super {
     preCheck = ''export PATH="$PWD/dist/build/ghcide:$PATH"'';
   }) super.ghcide;
 
+  # At least on 1.3.4 version on 32-bit architectures tasty requires
+  # unbounded-delays via .cabal file conditions. Let's pull it in
+  # unconditionally.
+  tasty = overrideCabal (drv: {
+    libraryHaskellDepends = (drv.libraryHaskellDepends or []) ++ [ self.unbounded-delays ];
```

This doesn't really improve CI to do it unconditionally.

```suggestion
    libraryHaskellDepends = (drv.libraryHaskellDepends or []) ++ lib.optionals stdenv.hostPlatform.is32bit [ self.unbounded-delays ];
```

or I guess

```suggestion
    libraryHaskellDepends = (drv.libraryHaskellDepends or []) ++ lib.optional (!(stdenv.hostPlatform.isAarch64 || stdenv.hostPlatform.x86_64)) [ self.unbounded-delays ];
```